### ISSUE TITLE
FEATURE REQUEST - Jump to running stitches

### ISSUE THREAD

**Issue by dragonlord66666** [inkstitch/inkstitch#1946]
I wonder if it would be possible to have an option were we can convert a jump into a running stitch
this way we could contour that running stitch around an outline and cut down on trims, would also be handy for those who don't have trims on their machine or, like mine, just ignores them even when it's meant to do trim's LOL

I know we can add running ourselves manually but would be handy/nice if the software could convent jumps into running stitches that we could then contour

**Comment by kaalleen** [comment]
Yes sure, nice idea. Here you go: https://github.com/inkstitch/inkstitch/releases/tag/dev-build-kaalleen-jump-to-stroke

It can be found int Extensions > Ink/Stitch > Tools: Stroke > Jump Stitch to Stroke

Just if you wonder about it: it only adds running stitch connections if the distance is moretan defined by your collapse length value in the inkstitch settings.

**Comment by dragonlord66666** [comment]
I did reply via email but guess they never got through. the pkg file would not install said  "inkstitch-kaalleen-jump-to-stroke-osx-pkg' can't be opened because Apple cannot check it for alicious software.  This software needs to be updated. contact the developer for more information" - normally I just go into security and allow but this did not have that option.
I managed to install the zip version but it fails every time I click an option - params or visualise for example I get the error Error loading Python lib '/Users/username/Library/Application Support/org.inkscape.Inkscape/config/inkscape/extensions/inkstitch/inkstitch.app/Contents/MacOS/libpython3.8.dylib': dlopen: dlopen(/Users/username/Library/Application Support/org.inkscape.Inkscape/config/inkscape/extensions/inkstitch/inkstitch.app/Contents/MacOS/libpython3.8.dylib, 10): Library not loaded: @loader_path/libintl.8.dylib
  Referenced from: /Users/username/Library/Application Support/org.inkscape.Inkscape/config/inkscape/extensions/inkstitch/inkstitch.app/Contents/MacOS/libpython3.8.dylib
  Reason: Incompatible library version: libpython3.8.dylib requires version 12.0.0 or later, but libintl.8.dylib provides version 11.0.0
  I have the latest version of python installed which is 3.11.1 on python.org
